frontend/src/pages/ramp-form/index.tsx

```diff
@@ -15,6 +15,7 @@ export const RampForm = () => {
   const activeSwapDirection = useRampDirection();
   const onSwapDirectionToggle = useRampDirectionToggle();
   const offrampKycStarted = useRampKycStarted();
+  const offrampKycLevel2Started = useRampKycLevel2Started();
```

Comment:
Let's move away from the `offramp` and embrace the `ramp`
```suggestion
  const rampKycStarted = useRampKycStarted();
  const rampKycLevel2Started = useRampKycLevel2Started();
```

Reply:
alright!